package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "nimble-survey",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@nimblehq/eslint-config-nimble": "2.1.1",
```

Comment:
The ESLint package already fixed regarding [this issue](https://github.com/nimblehq/react-templates/issues/26) on React template